uktrade/data-hub-frontend
# Add PieChart component

## Description of change

This is the `PieChart` component required for the new dashboard. It uses the [Nivo](https://nivo.rocks/pie/) library, which is a React wrapper for D3. This works perfectly in Storybook, but when adding into the skeleton it does not play nicely as I cannot get it to work with styled components. Due to these issues this PR is not ready for merge until they are resolved. Nivo does support `onClick` actions but I have not implemented these as I chose to focus on trying to fix the styling issues.

## Test instructions

With the `layoutTesting` feature flag enabled, go to the dashboard. If you have some investment projects set up you should see a pie chart instead of the pre-existing bar chart.

To see the expected behaviour, run Storybook and look at the new `PieChart` stories.

## Screenshots
<img width="404" alt="Screenshot 2021-03-04 at 17 25 03" src="https://user-images.githubusercontent.com/36161814/110005114-0adaeb00-7d10-11eb-9499-e63389f4af97.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
